vic/drivers/classic/src/write_data.c

```diff
@@ -222,4 +222,7 @@ write_data(out_data_file_struct *out_data_files,
             fprintf(out_data_files[file_idx].fh, "\n");
         }
     }
+    else {
+        log_err("Unrecognized OUT_FORMAT option");
```

Comment:
Leave this in - but we should never trigger this, right? Since we check the option during reading to the global file?